tests/modules/lint/test_module_lint_local.py

```diff
@@ -0,0 +1,53 @@
+import shutil
+from pathlib import Path
+
+import nf_core.modules.lint
+
+from ...test_modules import TestModules
+
+
+class TestModulesLintLocal(TestModules):
+    """Test ModuleLint functionality with local modules"""
+
+    def test_modules_lint_local(self):
+        """Test linting local modules"""
+        assert self.mods_install.install("trimgalore")
+        installed = Path(self.pipeline_dir, "modules", "nf-core", "trimgalore")
+        local = Path(self.pipeline_dir, "modules", "local", "trimgalore")
+        shutil.move(installed, local)
```

Comment:
same to samtools/sort before, these steps should be in the setup of this test class